src/main/java/BMO/task/Event.java

```diff
@@ -0,0 +1,76 @@
+package bmo.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task.
+ */
+public class Event extends Task {
+
+    private LocalDate from;
+    private LocalDate to;
+    private DateTimeFormatter outputFormatter;
+
+    /**
+     * Constructor for Event class.
+     *
+     * @param description Description of the task
+     *
+     * @param from Start date of the event
+     * @param to End date of the event
+     */
+    public Event(String description, String from, String to) {
+        super(description);
+
+        // Define input date format
+        DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+
+        // Convert string to  LocalDate objects
+        this.from = LocalDate.parse(from, inputFormatter);
+        this.to = LocalDate.parse(to, inputFormatter);
+
+        // Define output date format
+        DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("MMM-dd-yyyy");
+        this.outputFormatter = outputFormatter;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + this.getPrintedFrom() + " to: " + this.getPrintedTo() + ")";
+    }
+
+    /**
+     * Returns the start date in the format MMM-dd-yyyy to be printed in the console.
+     */
+    private String getPrintedFrom() {
+        return this.from.format(this.outputFormatter);
+    }
+
+    /**
+     * Returns the end date in the format MMM-dd-yyyy to be printed in the console.
+     */
+    private String getPrintedTo() {
+        return this.to.format(this.outputFormatter);
+    }
```

Comment:
i think the naming for this works well when used in the toString method. perhaps for greater clarity it could be getPrintedEndDate?